cpp/mg_utility/mgp.hpp

```diff
@@ -98,29 +101,40 @@ struct mgp_value *value_make_path(struct mgp_path *val) {
   return MgInvoke<mgp_value *>(mgp_value_make_path, val);
 }
 
-enum mgp_value_type value_get_type(struct mgp_value *val) {
-  return MgInvoke<enum mgp_value_type>(mgp_value_get_type, val);
+struct mgp_value *value_make_date(struct mgp_date *val) {
```

Comment:
As this file is a C++ file, it is unnecessary to write out all of the `struct` and `enum` prefixes for the parameter types.

Reply:
Pushed :heavy_check_mark: